Cargo.toml

```diff
@@ -13,7 +13,17 @@ members = [
 ]
 default-members = ["xtask"]
 exclude = ["zircon-user", "rboot"]
+resolver = "2"
 
 [profile.release]
 lto = true
 debug = true
+
+[workspace.dependencies]
+log = "0.4"
+riscv = "0.10.1"
+executor = { git = "https://github.com/KaminariOS/PreemptiveScheduler", rev = "c9299748f467cf55eb6f3c2248791cc93aaf9fc4" }
```

Comment:
What's changed compared with the current version?

```
executor = { git = "https://github.com/DeathWish5/PreemptiveScheduler", rev = "e8cd353" }
```

Reply:
I removed some feature flags that have been stabilized in the latest Rust version. I also opened PRs for the original repos.

Reply:
cool. cc @DeathWish5 please also take a look.